client/app/types/course/assessment/submission/logs.ts

```diff
@@ -0,0 +1,26 @@
+export interface LogsData {
+  validAttempt: boolean;
+  timestamp: Date;
```

Comment:
We get a string from the backend?

Reply:
has been reformatted. Actually it doesn't matter since the string is certainly of the Date format also from backend, but yeah I changed it into `string` for clarity purpose